src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_RANDOM_NUMBER = 100;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random ballRandomNumber = new Random();
+    private final int ballRandomNumberValue = ballRandomNumber.nextInt(MAX_RANDOM_NUMBER);
```

Comment:
don’t make this variable a class field.
The class fields should represent the state of the class objects otherwise they should be local variables.